tst/teststandard/weakptr.tst

```diff
@@ -77,4 +77,19 @@ false
 gap> Unbind(w[2]);
 gap> Print(w,"\n");
 WeakPointerObj( [ 1, , E(5), 311, , fail ] )
+gap> Immutable(w);
+[ 1,, E(5), 311,, fail ]
+gap> w;
+WeakPointerObj( [ 1, , E(5), 311, , fail ] )
+gap> MakeImmutable(w);
+[ 1,, E(5), 311,, fail ]
+gap> w;
+[ 1,, E(5), 311,, fail ]
+gap> w := WeakPointerObj([1,,2^40*10^10,,[2,3,4],fail,]);
+WeakPointerObj( [ 1, , 10995116277760000000000, , [ 2, 3, 4 ], fail ] )
+gap> MakeImmutable(w);
+[ 1,, 10995116277760000000000,, [ 2, 3, 4 ], fail ]
+gap> GASMAN("collect");
+gap> w;
+[ 1,, 10995116277760000000000,, [ 2, 3, 4 ], fail ]
```

Comment:
This test is specifically to check that calling `MakeImmutable` on weak pointer objs does correctly transform them into normal lists.